src/main/java/uk/gov/justice/digital/hmpps/cmd/api/repository/SnoozePreferenceRepository.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.cmd.api.uk.gov.justice.digital.hmpps.cmd.api.repository
+
+import org.springframework.data.repository.CrudRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.cmd.api.uk.gov.justice.digital.hmpps.cmd.api.model.SnoozePreference
+import java.time.LocalDate
+import java.util.*
+
+@Repository
+interface SnoozePreferenceRepository : CrudRepository<SnoozePreference, UUID> {
```

Comment:
Is the use of `SnoozePreference` and `SnoozePreferences` in various naming bits an intentional differentiation?

Reply:
I have renamed to userPreference consistently everywhere